Userland/Libraries/LibWeb/SVG/AttributeParser.cpp

```diff
@@ -230,7 +230,8 @@ void AttributeParser::parse_elliptical_arc()
 
 float AttributeParser::parse_length()
 {
-    return parse_sign() * parse_number();
+    // https://www.w3.org/TR/SVG11/types.html#DataTypeLength
+    return parse_number(true);
```

Comment:
Can we use a Yes/No enum instead? This is pretty opaque as-is.

Reply:
I think it might be better to have a `parse_nonnegative_number()` method instead?

Reply:
sure, that also works